Makefile

```
@@ -39,10 +39,12 @@ js:
 i18n:
 	$(PYTHON) ./scripts/i18n-messages compile
 
-git:
+git:	
+ifeq ($(DOCKER_HUB),FALSE)
```

I am sorry...  I think my comments were misunderstood.  Let's change this back to:
```suggestion
ifneq ($(DOCKER_HUB),true)
```